modules/virt-setting-mtv-lm-feature-gates.adoc

```diff
@@ -0,0 +1,41 @@
+// Module included in the following assemblies:
+//
+// * virt/live_migration/virt-enabling-cclm-for-vms.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="virt-setting-mtv-lm-feature-gates_{context}"]
+= Setting a live migration feature gate in the {mtv-first}
+
+Enable the {product-title} live migration feature gate in the {mtv-first} by modifying the `spec` stanza in the `forklift-controller` custom resource (CR).
+
+.Prerequisites
+
+* You have installed the {oc-first}.
+
+* You must have cluster admin privileges.
+
+* The `virt-synchronization-controller` pods must be running.
+
+.Procedure
+
+. To enable the feature gate by modifying the CR, run the following command:
++
+[source,terminal]
+----
+$ oc patch ForkliftController forklift-controller -n openshift-mtv --type json -p '[{"op": "add", "path": "/spec/feature_ocp_live_migration", "value": "true"}]'
+----
+
+.Verification
+
+* Verify that the feature gate is enabled by confirming that the `feature_ocp_live_migration` key value is set to `true`, as shown in the following example:
++
+[source,yaml]
+----
+# ...
+spec:
+  feature_ocp_live_migration: "true"
+  feature_ui_plugin: "true"
+  feature_validation: "true"
+  feature_volume_populator: "true"
+# ...
+----
```

Comment:
Is this in a CR somewhere, or in the web console, etc? Can we direct someone to _where_ they would check this, or what command they would run to see this output?

If it's in a CR, it needs to include things like the `apiVersion` and `kind` per https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#required-fields

Reply:
Yep that is correct.